### PR TITLE
WD-16847 Rebranded /observability/what-is-observability

### PR DESCRIPTION
**Pull request by muhammad-ali-pk**
## Done

- Rebranded page

## QA

- View the site  in your web browser at: https://ubuntu-com-14588.demos.haus/observability/what-is-observability
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check that it matches the following:
   - [Design](https://www.figma.com/design/BycwGsYlyy6sYR6oD60O76/25.04-Observability-Rebranding?node-id=1-5699&t=btTQoadAeJLMzT0A-0)
   - [CopyDoc](https://docs.google.com/document/d/1eoqDQzFiRJY2u_EO4JBuLBLqae4K3nrEf3qd88KHw0Y/edit)

## Issue / Card

Fixes #[WD-16847](https://warthogs.atlassian.net/browse/WD-16847)

[WD-16847]: https://warthogs.atlassian.net/browse/WD-16847?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Sophie-32** [comment]
- [ ] missing tags for decorative image since the images are explained in the paragraph
Screen Shot 2024-12-19 at 5 08 53 PM
Screen Shot 2024-12-19 at 5 09 41 PM

- [ ] fix broken link for
Screen Shot 2024-12-19 at 5 10 50 PM
should link to https://ubuntu.com/blog/tag/model-driven-operations

You can UX+1 with those changes

**Comment by mattea-turic** [comment]
Thanks @muhammad-ali-pk ! Three v small things:

For the hero section:
- [ ] I feel that the pargraph text is a little off; has there always been a shallow padding above?

For "Controllability: observability in action":
- [ ] Could you pls add a br after "controllability:"?

For "Ready to make your applications observable?"
- [ ] Could you also add a br here, after "make"?

**Comment by muhammad-ali-pk** [comment]
>  I feel that the pargraph text is a little off; has there always been a shallow padding above?
Thanks @mattea-turic . Yes, this hero is rendered using Vanilla patterns and it seems like it renders and empty padded block even if there is no subtitle_text/h2 above the description. I have communicated this to Vanilla team. But yeah, this has been that way everywhere where there's Hero pattern used.

**Comment by mattea-turic** [comment]
> Update: I have opened up a PR on vanilla for this. Once it gets approved and merged, this will automatically look like this

Great! I'll add a +1 for design then

File: templates/observability/what-is-observability.html [hunk 1]
```diff
@@ -1,251 +1,273 @@
...
+        </h2>
+      </div>
+      <div class="col">
+        <p>
+          Canonical can support your open source observability stack with security fixes and LTS support, as well as run your best-of-breed, open source monitoring tools reliably and at scale.
+        </p>
+        <div class="p-cta-block">
+          <a href="https://docs.google.com/document/u/0/d/1NxJdT3BD6EWnpsu-eathNyranQVImnDGhuCjzsIWh4s/edit"
+             class="p-button--positive"
+             aria-label="Learn more about open source observability">More about open source observability</a>
```
Comment (akbarkz) [inline templates/observability/what-is-observability.html:267]: In the copy doc the label of the button is different

File: templates/observability/what-is-observability.html [hunk 2]
```diff
@@ -1,251 +1,273 @@
...
+          <br />
+          your applications observable?
+        </h2>
+      </div>
+      <div class="col">
+        <p>
+          Canonical can support your open source observability stack with security fixes and LTS support, as well as run your best-of-breed, open source monitoring tools reliably and at scale.
+        </p>
+        <div class="p-cta-block">
+          <a href="https://docs.google.com/document/u/0/d/1NxJdT3BD6EWnpsu-eathNyranQVImnDGhuCjzsIWh4s/edit"
```
Comment (akbarkz) [inline templates/observability/what-is-observability.html:265]: This should link to the bubble's index page. In the copy doc it's pointing to the google doc too, but it's nonsense.

**Review by akbarkz — CHANGES_REQUESTED** [review]

File: templates/observability/what-is-observability.html (hunk 2 above)
Comment (muhammad-ali-pk) [reply templates/observability/what-is-observability.html]: Thanks for the correction! @akbarkz

**Review by akbarkz — APPROVED** [review]

**Pull request merged**